src/test/java/org/springframework/integration/aws/config/xml/SqsMessageDrivenChannelAdapterParserTests.java

```diff
@@ -85,6 +86,7 @@ public void testSqsMessageDrivenChannelAdapterParser() {
 		SimpleMessageListenerContainer listenerContainer =
 				TestUtils.getPropertyValue(this.sqsMessageDrivenChannelAdapter, "listenerContainer",
 						SimpleMessageListenerContainer.class);
+		listenerContainer.setAutoStartup(false);
```

Comment:
This is a hack which doesn't reflect the real picture.
Please, remove it and rely only on the XML config.
That's why your original fix must be present in the `SqsMessageDrivenChannelAdapter`.

Thanks